=== modified file 'extensions/BMO/template/en/default/hook/bug/show-header-end.html.tmpl'
--- extensions/BMO/template/en/default/hook/bug/show-header-end.html.tmpl	2012-11-30 15:53:02 +0000
+++ extensions/BMO/template/en/default/hook/bug/show-header-end.html.tmpl	2013-01-09 20:12:47 +0000
@@ -8,11 +8,8 @@
 
 [% style_urls.push('extensions/BMO/web/styles/edit_bug.css') %]
 [% javascript_urls.push('extensions/BMO/web/js/edit_bug.js') %]
-[% title = "$bug.bug_id &ndash; " %]
+[% title = "$terms.Bug $bug.bug_id &ndash; $filtered_desc" %]
 [% IF bug.alias != '' %]
   [% title = title _ "($bug.alias) " %]
 [% END %]
 [% title = title _ filtered_desc %]
-[% javascript = javascript _ 
-                "document.title = document.title.replace(/^" _ terms.Bug _ " /, '');"
-%]

